### ISSUE TITLE
Unable to compile

### ISSUE THREAD

**Issue by Sub2MrYeast** [immortius/chunkbychunk#139]
From what I can tell the Plugins that Forge needs to compile are for gradle versions 8.0+. But most functions in the modpack rely on old deprecated commands before 8.0. There's probably a way to fix it, but I am very new to all of this and don't know how. I've spent many hours on this and was hoping someone could help.

**Comment by immortius** [comment]
Yeah, this is annoying, some of the minecraft gradle plugins don't version correctly so roll out breaking changing like dependency on newer gradle versions. And the gradle wrapper update command doesn't work if the plugins are broken, The fix is:

1. In settings.gradle, remove the `include` section
2.  Run `./gradlew wrapper --gradle-version 8.12`
3. Restore the include section back to settings.gradle
4. ForgeGradle plugin will also need to be updated to `6.0.+` or later
5. I haven't checked the changes needed for NeoForge for the minecraft versions using it, but possibly similar.

Everything should work from there, or at least be easier to approach.

For reference, the way I was developing to support multiple minecraft versions was to make functional changes on the 1.18.2 branch, and then rebase upwards through the supported versions off of the previous version (e.g. rebase 1.19.2 off 1.18.2, then 1.19.3 off 1.19.2 and so forth), fixing issues as I went.

**Comment by Sub2MrYeast** [comment]
> 2\. ./gradlew wrapper --gradle-version 8.12

Worked like a charm. I really appreciate it. Love this mod, and I am really intrigued with the formula/script you used with the world mender to generate those chunks in that pattern. I know someone already commented this idea, but figuring out a way to use that formula with a command would be awesome. Maybe I can redneck engineer something.
